_source/_includes/legacy-api.html

```diff
@@ -0,0 +1,17 @@
+<div class="alert alert-info" role="alert">
+  <p class="title">
+    <a class="close" aria-label="Close" title="Close" onclick="this.parentElement.parentElement.style.display='none'">
+      <span aria-hidden="true">&times;</span>
+    </a>
+    Heads up... This blog post is old!
+  </p>
+  <p>This post is designed to work with the older Okta legacy API. If you're a
```

Comment:
The words "older" and "legacy" are kinda the same thing. It'd be cool if we could just say "v1" or something that doesn't imply it's bad.

Reply:
This is just temporary -- I'm going based on the official naming they're using in Product atm. Right now it's "legacy", but it's likely to change at some point. That's why this PR isn't updating any of the post metadata, just prepping it for later =)